network.go

```diff
@@ -611,6 +617,10 @@ func (n *network) driver(load bool) (driverapi.Driver, error) {
 }
 
 func (n *network) Delete() error {
+	return n.delete(false)
+}
+
+func (n *network) delete(force bool) error {
```

Comment:
Agree from the function semantic point of view a force delete is a force delete.
Changed `delete(..)`. PTAL.